### ISSUE TITLE
my agent does not learning

### ISSUE THREAD

**Issue by Hongsungchan** [Unity-Technologies/ml-agents#5738]
![Simple](https://user-images.githubusercontent.com/43347736/165652945-220a3c09-041a-4c82-ba89-4979e547dfac.png)

I am trying to train the agent by making a simple example(agent move to ball), but the agent training does not proceed properly.

Learning proceeds, but the agent is very dumb.

Even after learning is over, it does not move to the ball and is stuck in the corner.

Also, the generated model file cannot be used.

An error message appears in the command prompt when learning is in progress, and this seems to be the cause.

![image](https://user-images.githubusercontent.com/43347736/165653218-7caa501c-077d-49eb-9b36-e8e4238e6fcb.png)

C:\Users\홍성찬\AppData\Local\Programs\Python\Python37\lib\site-packages\mlagents\trainers\torch\networks.py:91: UserWarning: Creating a tensor from a list of numpy.ndarrays is extremely slow. Please consider converting the list to a single numpy.ndarray with numpy.array() before converting to a tensor. (Triggered internally at  C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\torch\csrc\utils\tensor_new.cpp:210.)
  enc.update_normalization(torch.as_tensor(vec_input))

C:\Users\홍성찬\AppData\Local\Programs\Python\Python37\lib\site-packages\mlagents\trainers\torch\utils.py:320: UserWarning: The use of `x.T` on tensors of dimension other than 2 to reverse their shape is deprecated and it will throw an error in a future release. Consider `x.mT` to transpose batches of matricesor `x.permute(*torch.arange(x.ndim - 1, -1, -1))` to reverse the dimensions of a tensor. (Triggered internally at  C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\TensorShape.cpp:2318.)
  return (tensor.T * masks).sum() / torch.clamp(

![image](https://user-images.githubusercontent.com/43347736/165653864-16d3142d-8626-4cd9-9dcc-d24359efcc3c.png)

Looks like a python problem, do i need to reinstall python?

**Comment by miguelalonsojr** [comment]
We are now using the [Unity forums](https://forum.unity.com/forums/ml-agents.453/) to handle [discussions, installation issues, etc] like this. Please create a new thread there, in order to get the community involved in the discussion, and reply with the link to your forum thread.
